YouGoJoe/prodigy-backbone-practice
# Basic Todo app built in BackboneJS

## Description
Created the Todo app with minimal functionality as practice with BackboneJS. Users have the ability to create Todos, delete Todos, and mark Todos as complete (and incomplete). The app always starts with the same two Todos ("My First Todo" and "The other Todo").

## How Has This Been Tested?
I have created Todos, deleted Todos using the 'X' button, and marked Todos as complete and incomplete using the checkbox. All changes except deletion persist to LocalStorage (why the deletion isn't being persisted is going to be investigated and fixed before the final commit).

## How Should This Be Manually Tested?
Users should create a Todo, delete a Todo, mark a Todo as complete, and mark a Todo that was complete back to incomplete. All changes can be observed in LocalStorage (except deletion, mentioned previously).

## Screenshots (if appropriate):

## Types of changes


## Checklist:


- [ ] My change requires a change to the documentation, which I have included.
- [ ] This requires some SQL scripts to be run, which I included in a "migrations" folder
